src/jvmMain/kotlin/io/beatmaps/api/playlist.kt

```diff
@@ -214,34 +238,49 @@ fun Route.playlistRoute() {
         val searchFields = PgConcat(" ", Playlist.name, Playlist.description)
         val searchInfo = parseSearchQuery(it.q, searchFields)
         val actualSortOrder = searchInfo.validateSearchOrder(it.sortOrder)
+        val sortArgs = when (actualSortOrder) {
+            SearchOrder.Relevance -> listOf(searchInfo.similarRank to SortOrder.DESC, Playlist.createdAt to SortOrder.DESC)
+            SearchOrder.Rating, SearchOrder.Latest -> listOf(Playlist.createdAt to SortOrder.DESC)
+            SearchOrder.Curated -> listOf(Playlist.curatedAt to SortOrder.DESC_NULLS_LAST, Playlist.createdAt to SortOrder.DESC)
+        }.toTypedArray()
 
         newSuspendedTransaction {
             val playlists = Playlist
+                .joinMaps(type = JoinType.LEFT) {
```

Comment:
This is the default and `Beatmap.deletedAt.isNull()` should possibly be the default for state